conftest.py

```diff
@@ -0,0 +1,19 @@
+"""Customize pytest configuration"""
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pytest import Config, Parser
```

Comment:
Out of curiosity, what makes you go all-in on `TYPE_CHECKING`?

I usually only use it to accommodate lazy imports, optional dependencies, or circular imports

Reply:
No real reason--just imitating the style of some examples (I unfortunately forget which ones at this point). I'm totally open to dialing it back.